.pre-commit-config.yaml

```diff
@@ -1,5 +1,5 @@
 default_language_version:
-  python: "3"
+  python: "3.12"
```

Comment:
We want to leave the version at 3.12 to avoid having to have the odd dependency setup for 3.13.  Once psycopg ships with a binary, we can change this to 3.13